docs/agent/PR_BOT_POLICY.md

```
@@ -0,0 +1,40 @@
+# PR Bot Policy
+
+## Goals
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)